tests/aws/services/secretsmanager/test_secretsmanager.py

```diff
@@ -261,6 +261,125 @@ def test_call_lists_secrets_multiple_times_snapshots(
         )
         sm_snapshot.match("delete_secret_res_1", delete_secret_res_1)
 
+    @markers.aws.validated
+    def test_list_secrets_filtering(self, aws_client, create_secret):
```

Comment:
Using static names / identifiers can be problematic when this test is executed multiple times (retries?).
Also, humans are remarkably un-creative when it comes to naming (the next person writing a test will most likely create a similar identifier).
This is why the best practice would be to use random names for every test execution.
You could f.e. use `from localstack.utils.strings import short_uid`...

Do you think it would be feasible to create randomized identifiers in this test case without blowing up the complexity of the filter tests?